nav/nav_model/nav_model_stars.py

```
@@ -740,13 +743,14 @@ def _create_model(self,
         self._stretch_regions = stretch_regions
         self._metadata = metadata
 
-        self.logger.debug(f'  Star model min: {np.min(self._model_img)}, max: {np.max(self._model_img)}')
+        self.logger.debug(f'  Star model min: {np.min(self._model_img)}, '
+                          f'max: {np.max(self._model_img)}')
```

Ruff (0.14.7)

Logging statement uses f-string

(G004)